operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/util/BulkDataExportUtil.java

```diff
@@ -103,7 +103,7 @@ public void checkExportPatientResourceTypes(List<String> resourceTypes) throws F
     public Set<String> getSupportedResourceTypes() {
         try {
             List<String> rts = FHIRConfigHelper.getSupportedResourceTypes();
```

Comment:
as paul alludes to in the PR description `FHIRConfigHelper.getSupportedResourceTypes()` is much improved in the `r4b` branch.  in the target branch of this PR, `main`, it just reads the resources config without considering whether the server is open (supports all types) or closed (supports only the listed ones).

I'm not sure if the logic below handles all the cases properly, but is the idea to wait on the updated handling from `FHIRConfigHelper.getSupportedResourceTypes()` for that?

Reply:
if so, I think thats fine with me but maybe we should leave a note somewhere..maybe in #3083 ?

Reply:
That's right, since the adapter is there, I think that's the long term answer. Rather than recreate in main